lib/Conversion/TorchToTosa/TorchToTosa.cpp

```diff
@@ -3784,34 +3797,36 @@ LogicalResult ConvertAtenOp<Aten_IndexPutImplOp>::matchAndRewrite(
     return rewriter.notifyMatchFailure(
         op, "Expected indices to have same shape as values");
 
-  
   auto outType =
       dyn_cast<RankedTensorType>(getTypeConverter()->convertType(op.getType()));
   if (!outType)
     return rewriter.notifyMatchFailure(
         op, "Only tensor types input are currently supported");
 
-
   auto numInElements = self.getType().getShape()[0];
   auto numValues = values.getType().getShape()[0];
 
   // TOSA scatter requires 3d in and 2d indices & values
-  SmallVector<int64_t> scatterInOutShape {1, numInElements, 1};
-  SmallVector<int64_t> scatterIndicesShape {1, numValues};
-  SmallVector<int64_t> scatterInputShape {1, numValues, 1};
+  SmallVector<int64_t> scatterInOutShape{1, numInElements, 1};
```

Comment:
Did you run some kind of formatter? It feels like there are a lot of those changes

Reply:
Hm, it was supposed to only format the lines I modified. I will revert the rest.